src/main.cpp

```diff
@@ -43,10 +43,8 @@ void setup()
 
     // Initialize serial output
     Serial.begin(SERIAL_BAUDRATE);
-#if ARDUINO_USB_CDC_ON_BOOT
-    Serial.setTxTimeoutMs(0);
-    delay(100);
-#else
+#if !ARDUINO_USB_CDC_ON_BOOT
+    // Only wait for serial interface to be set up when not using CDC
```

Comment:
Espressif's Documentation of [USB CDC (Communications Device Class)](https://docs.espressif.com/projects/arduino-esp32/en/latest/api/usb_cdc.html) and the respective `event`'s.

```c++
#if !ARDUINO_USB_CDC_ON_BOOT
USBCDC USBSerial;
#endif
```